src/features/home/pages/Home.tsx

```diff
@@ -78,7 +79,7 @@ export const Home: FunctionComponent = () => {
 
   useEffect(() => {
     const displaySubscriptionModal = async () => {
-      if (!isLoggedIn) return
+      if (!isLoggedIn || user?.status.statusType === YoungStatusType.non_eligible) return
```

Comment:
J'espère qu'on aura pas l'erreur  `cannot read property statusType of undefined`, on avait un problème en prod sur cette propriété je pense